app/prometheus/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM prom/prometheus
+ADD prometheus.yml /etc/prometheus/
+ADD entrypoint.sh /etc/prometheus
```

Comment:
Inconsistency in trailing slashes

Reply:
Also, I'd add some extra spaces for formatting here, up to you

Reply:
added newlines and removed first trailing slash